src/create-probot-app.ts

```diff
@@ -195,11 +207,20 @@ async function main(): Promise<void> {
 
   sanitizeBy(answers, ["author", "description"]);
 
-  const relativePath = path.join(templatePath, answers.template);
-  const generateResult = await generate(relativePath, destination, answers, {
+  // TODO: add logic to properly handle path conflicts between __common__ and actual
+  // template folders
```

Comment:
could you clarify the comment? What is currently happening if there is a file name conflict, what would you like to happen?

Reply:
With the proposed code and folder structure change, behavior will be the same as for current `master` as far as no conflicting names between `__common__` and the templates are introduced. In case of conflicts, template file path won't overwrite `__common__` path unless overwrite is forced via CLI arguments.

To make this right requires to rethink how the file list is generated and I thought better to skip that for now and understand if the change is going in the right direction or not.

Reply:
Properly addressed by https://github.com/probot/create-probot-app/commit/a74c6cb98521632051477fe2ba98b82590504174